task_DEV_1/task_DEV_1/Program.cs

```diff
@@ -7,23 +7,28 @@ class Program
         static void Main(string[] args)
         {
             Console.WriteLine("Enter A number");
-            int inputNumber = int.Parse(Console.ReadLine());
+            int firstNumber = int.Parse(Console.ReadLine());
+            Console.WriteLine("Enter B number");
+            int lastNumber = int.Parse(Console.ReadLine());
 
-            if (inputNumber % 2 == 0 && inputNumber % 5 == 0)
+            for (; firstNumber <= lastNumber; firstNumber++)
```

Comment:
I would recommend you don't change inputNumber. Because in such case you will have different value of one variable at the begining and at the end of your code and it can confuse somebody who will use your code. For example it the end of your code I want to write to console initial values of input parameters. How can I do this? Feel free to ask question if my explanation is not complitly clear.

Reply:
How can I fix it?

Reply:
Without firstNumber ++ does not work

Reply:
You could create some helper temporary variable.